### DISCUSSION TITLE
Set ARM to save audo CDs to /music/ARTIST/ALBUM

### DISCUSSION THREAD

**Discussion by valemaio2** [automatic-ripping-machine/automatic-ripping-machine#978] [Q&A]
Hello all,
I'm trying to integrate ARM with Lidarr; the way Lidarr recognizes music is by using the folder structure:

`/music/ARTIST/ALBUM/`

while ARM saves audio CDs with a folder structure of:

`/music/ARTIST - ALBUM/`

I'm sure there is a way to set abcde to use the /music/ARTIST/ALBUM/ folder structure, but I couldn't find it, and a search in the discussion section of this GitHub yold no result. Could anyone please enlighten me? Thank you very much.

**Comment by 1337-server** [comment]
The basics are covered here and in the manual: https://github.com/automatic-ripping-machine/automatic-ripping-machine/wiki/.abcde.conf

**Comment by meeprophone** [comment]
This is governed by the 'abcde' subsystem. Edit your config/abcde.conf file, something like:

`OUTPUTFORMAT='${ARTISTFILE}/${ALBUMFILE}/${ARTISTFILE} - ${ALBUMFILE} - ${TRACKNUM} - ${TRACKFILE}'`

The available placeholders are listed in the file as a comment.
